VeroliBGV/BGV008.xml

```diff
@@ -0,0 +1,836 @@
+                         <note>See the transcription in <ref target="#LIT4009Naahab"/>.</note>
+                      </msItem>
+                     <msItem xml:id="ms_i1.2">
+                        <locus from="3r" to="136r"/>
+                        <title type="complete" ref="LIT2000Mazmur"/>
+                        <incipit xml:lang="gez">
+                           <locus target="#3ra"/>
+                           <lb/><hi rend="rubric">ፍካሬ፡ ዘጻድቃን፡ ወዘኃጥአን።</hi>
+                           <lb/>መዝሙር፡ ዘዳዊት። <hi rend="rubric">ሃሌ፡ ሉያ።
+                              <lb/>፩ ብፁዕ፡ ብእሲ፡ ዘኢሖረ፡ በምክረ፡ ረሲአን።</hi> 
+                           <lb/>ወዘኢቆመ፡ ውስተ፡ ፍኖተ፡ ኃጥአን።
+                        </incipit>
+                        <explicit xml:lang="gez">
+                           <locus target="#136r"/>
+                           <lb/>ወረገመኒ፡ በአማልክቲሁ።
+                           <lb/>ወአንሰ፡ ነሣእኩ፡ ዘእምላዕሌሁ፡ ሰይፈ።
+                           <lb/>ወመተርኩ፡ ርእሶ።
+                           <lb/>ወአሰሰልኩ፡ ጽእለተ፡ እምደቂቀ፡ ፳ኤል።
+                        </explicit>
+                     </msItem>
+                     
+                     <msItem xml:id="ms_i1.3">
+                        <locus from="136r" to="150r"/>
+                        <title type="complete" ref="LIT1828Mahale"/>
+                        <msItem xml:id="ms_i1.3.1">
+                           <locus from="136r" to="137v"/> 
+                           <title type="complete" ref="LIT1828Mahale#Moses1"/>
+                           <note/>
+                           <incipit xml:lang="gez">
+                              <locus target="#136r"/>
+                              <lb/><hi rend="rubric">መሐልየ፡ ነቢያት፡ ወጸሎቱ፡ ለሙሴ።</hi>
+                              <lb/>ንሴብሖ፡ እግዚአብሔር፡ ስቡሐ፡ ዘተሰብሖ።
+                              <lb/><hi rend="rubric">ፈረሰ፡ ወመስተፅዕነ፡ ወረወ፡ ውስተ፡ ባሕ<supplied reason="omitted">ር</supplied>።</hi>
+                           </incipit>
+                           <explicit xml:lang="gez">
+                              <locus from="137r" to="137v"/>
+                              <lb/>ወአስተጋብአ፡ እግዚአብሔር፡ ላዕሌሆሙ፡ ማ<add place="below">የ፡ ባሕር።</add>
+                              <pb n="137v"/>
+                              <lb/>ወደቂቀ፡ ፳ኤልሰ፡ <add place="above">ኃለፉ፡</add> እንተ፡ የብስ፡ በማዕከለ፡ ባሕ<add place="above">ር</add>። 
+                              <lb/>ማይሰ፡ ሎሙ፡ ዓረፍት፡ በይምን፡ ወዓረፍት፡ በፅ<add place="below">ግም።</add>
+                           </explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.3.2">
+                           <locus from="137v" to="139r"/>
+                           <title type="complete" ref="LIT1828Mahale#Moses2"/>
+                           <note/>
+                           <incipit xml:lang="gez">
+                              <locus target="#137v"/>
+                              <lb/><hi rend="rubric">ጸ<subst><del rend="erasure" unit="chars" quantity="1"/><add place="overstrike">ሎ</add></subst>ተ፡ ሙሴ፡ ዘዳግም፡ ሕግ።
```

Comment:
I would suggest to distinguish them when it is possible.

Reply:
we can distinguish them as far as we encode this information for BM and possibly merge the two cases in the printed catalogue (both "effaced" and "erased" generate the same output). But since BM allows both possibilities I would keep distinguishing them.